docs/data-management/metrics/ratio-metric.md

```diff
@@ -3,15 +3,54 @@ sidebar_position: 2
 ---
 
 # Ratio metrics
+## Overview
+Ratio metrics allow you to calculate the ratio of two [simple metrics](/data-management/metrics/simple-metric). The most common use case for this is when the randomization entity is different from the analysis entity. 
+For example, consider the metric _average  order value_. Typically, experiments are not randomized at the _order_ level because a user can have multiple orders. 
```

Comment:
```suggestion
For example, consider the metric _average order value_. Typically, experiments are not randomized at the _order_ level because a user can have multiple orders. 
```